src/config.rs

```diff
@@ -111,7 +216,7 @@ macro_rules! build_cf_opt {
     }};
 }
 
-cf_config!(DefaultCfConfig);
+cf_config!(DefaultCfConfig, test_default_cf_config);
```

Comment:
can we use a cf_config_test macro instead? 
It is strange to put them together in one macro.